packages/components/src/SvgIcon/utils/createSvgIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import type { IconProps } from "../SvgIcon";
+import React from "react";
+import SvgIcon from "../SvgIcon";
+
+/**
+ * Generates SVG Icons from paths and fragments.
+ *
+ * Implementation note: this is an extremely light wrapper right now,
+ * but may be helpful in the future -- for example, if we need to support
+ * forwarding refs or memoization across all Icons.
+ * This also makes it easier to directly grab icon files from Material UI.
+ */
+export default function createSvgIcon(path: React.ReactNode) {
```

Comment:
Traject is fully using the Material UI wrapper ([mui source code](https://github.com/mui-org/material-ui/blob/53a1655143aa4ec36c29a6063ccdf89c48a74bfd/packages/material-ui/src/utils/createSvgIcon.js), [our attempt at moving the file over earlier](https://github.com/chanzuckerberg/edu-design-system/pull/509/files/f6c45a035e95845f9ecf1f5aa6e7ca45e2c9fb6c#diff-c090b9da4c47a28b5f629a05251a93ad0d3b0a1ec4a95bb61bbd77954ea984a6))

@ahuth pointed out earlier that we're not using any ref forwarding currently, and it also may be overkill to memoize an Icon. I agree! but still think this abstraction is useful to keep around because:
1. It makes copy/pasting directly from MUI's icon components easier
2. If we end up needing forwarding, we can easily add it to all icons through this

Reply:
& for reference, here's what all the MUI components look like:
```
import * as React from 'react';
import createSvgIcon from './utils/createSvgIcon';

export default createSvgIcon(
  <path d="M21 11h-3.17l2.54-2.54c.39-.39.39-1.02 0-1.41-.39-.39-1.03-.39-1.42 0L15 11h-2V9l3.95-3.95c.39-.39.39-1.03 0-1.42a.9959.9959 0 00-1.41 0L13 6.17V3c0-.55-.45-1-1-1s-1 .45-1 1v3.17L8.46 3.63a.9959.9959 0 00-1.41 0c-.39.39-.39 1.03 0 1.42L11 9v2H9L5.05 7.05c-.39-.39-1.03-.39-1.42 0-.39.39-.39 1.02 0 1.41L6.17 11H3c-.55 0-1 .45-1 1s.45 1 1 1h3.17l-2.54 2.54c-.39.39-.39 1.02 0 1.41.39.39 1.03.39 1.42 0L9 13h2v2l-3.95 3.95c-.39.39-.39 1.03 0 1.42.39.39 1.02.39 1.41 0L11 17.83V21c0 .55.45 1 1 1s1-.45 1-1v-3.17l2.54 2.54c.39.39 1.02.39 1.41 0 .39-.39.39-1.03 0-1.42L13 15v-2h2l3.95 3.95c.39.39 1.03.39 1.42 0 .39-.39.39-1.02 0-1.41L17.83 13H21c.55 0 1-.45 1-1s-.45-1-1-1z" />
, 'AcUnitRounded');
```